src/components/trade/PriceImpact/usePriceImpact.ts

```diff
@@ -8,7 +8,17 @@ import { calculatePriceImpact, determinePriceWarning, getImpactColourClass } fro
 import { UsePriceImpactParams, UsePriceImpactReturn } from './types'
 
 function usePriceImpact(params: UsePriceImpactParams): UsePriceImpactReturn {
-  const { networkId, baseTokenId, quoteTokenId, limitPrice, fillPrice } = params
+  const {
+    networkId,
+    baseToken: { id: baseTokenId, decimals: baseTokenDecimals },
+    quoteToken: { id: quoteTokenId, decimals: quoteTokenDecimals },
+    limitPrice,
+    fillPrice: preFillPrice,
+  } = params
+
+  // Format fill price to quoteToken decimals
+  // Facilitates comparing limit/fill price
+  const fillPrice = preFillPrice?.decimalPlaces(quoteTokenDecimals, 1) || null
```

Comment:
not magic number. BigNumber has a constant for this

Reply:
The value 1 is correct, just use https://mikemcl.github.io/bignumber.js/#round-down

Reply:
:+1: